docs/archive.md

```diff
@@ -0,0 +1,26 @@
+# Data Archiving
+
+When the crawls are complete, the data is archived to several places:
+
+- Internet Archive (sql)
```

Comment:
These are just the data formats. The following sections will go into more detail.